pkg/controllers/user/helm/common.go

```diff
@@ -66,7 +67,16 @@ func helmInstall(tempDirs *common.HelmPath, app *v3.App) error {
 			}
 		}()
 	}
-	return common.InstallCharts(tempDirs, addr, app)
+	err := common.InstallCharts(tempDirs, addr, app)
+	if err != nil {
+		return "", err
+	}
+	// rendering notes here so tiller does not need to be started twice
+	notes, err := renderNotes(app, tempDirs, addr)
```

Comment:
This can be returned then the error check isn't needed

Reply:
now returning renderNotes and not doing error check